templates/index.html

```diff
@@ -272,25 +272,7 @@ <h4 class="title"></h4>
             <p class="mainStreet" style="margin:0"></p>
             <p class="secondaryStreet" style="margin:0"></p>
             <p class="junction" style="margin:0"></p>
-            <p class="involved_type" style="margin:0"></p>
```

Comment:
@danielhers Since the deleted fields belong to 'involved' and 'vehicles' data, a marker may contain several blocks of each. Hence the jQuery `.append` and the removal of the above fields